src/types/custom.d.ts

```diff
@@ -1 +1,2 @@
 declare module 'block_io';
+declare module 'coinstring';
```

Comment:
do we need it if we have `@types/coinstring`

Reply:
removed